### PR TITLE
Converted contributors.md into a table supporting GitHub and WordPress.org usernames

### PR DESCRIPTION
**Pull request by chetansatasiya**
Hi Team,

I have converted contributors.md into a table supporting GitHub and WordPress.org usernames.

Please review it and let me know if any changes on that.

Fixies: #172 

Thanks,
Chetan

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,3 @@
 .DS_Store
+.idea
+.idea/*
```
Comment (ntwb) [inline .gitignore]: Please remove these, these PHPStorm specific files should be ignored from a global `.gitignore` file in your own home directory, see https://stackoverflow.com/a/7335487/1911294 for how to do that.

File: CONTRIBUTORS.md [hunk 2]
```diff
@@ -1,14 +1,18 @@
+# Contributors
+
 WordPress.org Contributor Usernames
```
Comment (ntwb) [inline CONTRIBUTORS.md]: Updating the above would be great, probably remove this line altogether and change the title on line 1 to `Contributor Usernames`

File: CONTRIBUTORS.md [hunk 3]
```diff
@@ -1,14 +1,18 @@
...
+| @allancole | |
+| @karmatosed | |
+| @kjellr | |
+| @yingling017 | |
+| @mrasharirfan | |
+| @milana_cap | |
+| @fabiankaegy | |
+| @westonruter | |
+| @aaronjorbin | |
+| @netweb | |
```
Comment (ntwb) [inline CONTRIBUTORS.md]:
```suggestion
| @ntwb | @netweb |
```

Comment (ntwb) [reply CONTRIBUTORS.md]: The above fixes my usernames 🎉

File: .gitignore (hunk 1 above)
Comment (chetansatasiya) [reply .gitignore]: :+1:

File: CONTRIBUTORS.md (hunk 2 above)
Comment (chetansatasiya) [reply CONTRIBUTORS.md]: :+1:

File: CONTRIBUTORS.md (hunk 3 above)
Comment (chetansatasiya) [reply CONTRIBUTORS.md]: :+1:

**Comment by chetansatasiya** [comment]
Hi @ntwb 

Thanks for the quick look added the change.

Thanks,
Chetan

File: CONTRIBUTORS.md [hunk 4]
```diff
@@ -1,14 +1,16 @@
-WordPress.org Contributor Usernames
+# Contributors Usernames
```
[the remark this answers is not in the document]
Comment (chetansatasiya) [reply CONTRIBUTORS.md]: :+1:

**Comment by kjellr** [comment]
👋 It looks like the list was already converted to a table at some point recently, so I'm going to close this PR. I went ahead and added you to contributors.md in #409. Thanks for contributing!

**Pull request closed without merging**